src/com/makina/security/OpenFIPS201/ECParamsP256.java

```diff
@@ -33,7 +33,7 @@
 final class ECParamsP256 extends ECParams {
 
   // Curve polynomial element a
-  private static final byte[] a = {
+  private static byte[] a = {
```

Comment:
Why wouldn't these be final?

Reply:
Looking at the Google style guide,

"Constants are static final fields whose contents are deeply immutable and whose methods have no detectable side effects. Examples include primitives, strings, immutable value classes, and anything set to null. If any of the instance's observable state can change, it is not a constant. Merely intending to never mutate the object is not enough."

It's static, and I'd think it's a constant with a deeply immutable nature.

Reply:
This is an old pull request that is too far de-sync'd with the master branch now. You can see in the [head revision](https://github.com/makinako/OpenFIPS201/blob/1cdc610ab91379801eedaf804139af82a1d4c383/src/com/makina/security/openfips201/ECParamsP256.java#L36) these are all private static final.